pkg/fixtures/writer_mysql_plain.go

```diff
@@ -19,21 +19,38 @@ type mysqlPlainFixtureWriter struct {
 	logger   mon.Logger
 	dbClient db.Client
 	metaData *MysqlPlainMetaData
+	purger   *mysqlPurger
 }
 
 func MysqlPlainFixtureWriterFactory(metaData *MysqlPlainMetaData) FixtureWriterFactory {
 	return func(config cfg.Config, logger mon.Logger) FixtureWriter {
 		dbClient := db.NewClient(config, logger)
-		return NewMysqlPlainFixtureWriterWithInterfaces(logger, dbClient, metaData)
+		purger := newMysqlPurger(config, logger, metaData.TableName)
+
+		return NewMysqlPlainFixtureWriterWithInterfaces(logger, dbClient, metaData, purger)
 	}
 }
 
-func NewMysqlPlainFixtureWriterWithInterfaces(logger mon.Logger, dbClient db.Client, metaData *MysqlPlainMetaData) FixtureWriter {
+func NewMysqlPlainFixtureWriterWithInterfaces(logger mon.Logger, dbClient db.Client, metaData *MysqlPlainMetaData, purger *mysqlPurger) FixtureWriter {
```

Comment:
make this private to be consistent with the other writer.

Reply:
but then we can not unit test that, right?